component/src/main/java/org/wso2/extension/siddhi/gpl/execution/pmml/PmmlModelProcessor.java

```diff
@@ -182,7 +183,12 @@ protected void process(ComplexEventChunk<StreamEvent> streamEventChunk, Processo
                 default:
                     break;
             }
-            inData.put(inputField.getName(), inputField.prepare(String.valueOf(dataValue)));
+            try {
+                inData.put(inputField.getName(), inputField.prepare(String.valueOf(dataValue)));
+            } catch (InvalidResultException e) {
+                logger.error(String.format("Incompatible value for field: %s. Prediction might be erroneous.",
+                        inputField.getName()));
```

Comment:
better to include the actual exception message (InvalidResultException e) as well, as you only logger errors.

Reply:
Not better to include. You SHOULD include error trace.

Reply:
Fixed in commit [#82955b8dc64c8ad2fb9bb41b8c0be55ecfb67b3a](https://github.com/wso2-extensions/siddhi-gpl-execution-pmml/pull/32/commits/82955b8dc64c8ad2fb9bb41b8c0be55ecfb67b3a)